abstractions/python/kiota/abstractions/api_client_builder.py

```diff
@@ -0,0 +1,7 @@
+from .serialization import (
+    ParseNodeFactory,
+    ParseNodeFactoryRegistry,
+    SerializationWriterFactory,
+    SerializationWriterFactoryRegistry,
+)
+from .store import BackingStoreParseNodeFactory, BackingStoreSerializationWriterProxyFactory
```

Comment:
registration methods for default factories are missing

Reply:
Added in https://github.com/microsoft/kiota/pull/1060/commits/bef21f0ef341e93589ff2e66aa319da81485cdba